### ISSUE TITLE
JSS article doesn't show prompt

### ISSUE THREAD

**Issue by EDiLD** [rstudio/rticles#61]
The prompt for the JSS article is not showing.

I added

```{r}
knitr::opts_chunk$set(prompt = TRUE)
options(prompt = 'R> ')
```
to the top of my document. But this does not give the prompt.

However, building via
```{r}
rmarkdown::render('ms.Rmd', rmarkdown::md_document())
rmarkdown::render('ms.md', rmarkdown::pdf_document())
```

works.

I think this has something to to how RStudio builds the pdf.
Nevertheless, 

```{r}
options(prompt = 'R> ')
```

might be also added to `jss_article/skeleton/skeleton.Rmd`

**Comment by AngelBerihuete** [comment]
Hi EDiLD, maybe the following post could help you:

(http://stackoverflow.com/questions/38378527/jss-template-in-rticles-do-not-work-properly)

Now, when you update the rticles package it seems the prompt issue is gone.

**Comment by yihui** [comment]
I think it has been fixed in #78.
